algorithms/deathSpectrum/Valid_Parentheses.java

```diff
@@ -0,0 +1,55 @@
+package EjercicioAlgoritmos;
+
+import java.util.Scanner;
+
+public class Valid_Parentheses {
+
+		public static void parenthesis_pair(String s) {
+			//Inicializacion de variables
+			int size = s.length();
+			int iter = 0;
+			int tipo1=0, tipo2=0, tipo3=0;
```

Comment:
the variable names are a little confusing. To follow clean code practices is better to write the variables with a meaningful name.